### ISSUE TITLE
Menu dosen't open

### ISSUE THREAD

**Issue by Nat392** [Muqsit/InvMenu#224]
Hello,

Despite your fix, the bug is still there.

In the video, I open the interface, I quit, and I can't reopen it. 

https://github.com/Muqsit/InvMenu/assets/54687432/b8752850-5127-47e1-9a48-73a6ee73d27c

**Comment by Muqsit** [comment]
Minimal test case is working just fine:
```php
<?php

declare(strict_types=1);

use muqsit\invmenu\InvMenu;
use pocketmine\Server;

$server = Server::getInstance();
$player = $server->getPlayerExact("NeedleGalaxy");

InvMenu::create(InvMenu::TYPE_CHEST)->send($player);
```

**Comment by Nat392** [comment]
Bizzare, I've just tried again by deleting and updating InvMenu again, I've noticed a block with the "bug" texture appearing behind it, and all you have to do is right-click and it disappears.

https://github.com/Muqsit/InvMenu/assets/54687432/4391e732-8e21-4b03-85d4-50547255e728

**Comment by Nat392** [comment]
And I do use the latest version, safe and certain of that. I have the latest fix:

![image](https://github.com/Muqsit/InvMenu/assets/54687432/0029ae70-a50c-4460-9bd2-af44ee846246)

**Comment by Nat392** [comment]
I would like to sincerely apologize for the inconvenience I caused by informing you that I had found a bug in your plugin. After further investigation, I realized that the real source of the problem was actually external code written by a member of my team, which interfered with your plugin.

It turns out that a class from another plugin, totally independent of yours, was conflicting with InvMenu, creating this problematic situation.

Sorry again for the inconvenience.

Have a nice day!
